Healthy/Classes/Extensions/UITextField+Style.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+// MARK: - TextField style
+//
+extension UITextField {
+    enum TextFieldStyle {
+        case primary
+    }
+}
+
+// MARK: - Apply textfield style
+//
+extension UITextField {
+    func applyTextFieldStyle(_ style: TextFieldStyle) {
+        NSLayoutConstraint.activate([
+            heightAnchor.constraint(equalToConstant: Constants.defaultHeight),
+        ])
+        
+        layer.cornerRadius = Constants.defaultCornerRadius
+        layer.borderWidth = Constants.defaultBorderWidth
+        layer.borderColor = Constants.defaultBorderColor.cgColor
+        
+        setLeftPaddingPoints()
+        setRightPaddingPoints()
```

Comment:
Why?

Reply:
because text field in design have padding. so i added this functions.

Reply:
Please document this as a workaround 
```suggestion
        // Workaround for adding leading & training spaces
        setLeftPaddingPoints()
        setRightPaddingPoints()
```